libass/ass.c

```diff
@@ -332,51 +467,40 @@ static int process_event_tail(ASS_Track *track, ASS_Event *event,
                               char *str, int n_ignored)
 {
     char *token;
-    char *tname;
     char *p = str;
-    int i;
     ASS_Event *target = event;
+    size_t tokenpos = 0;
+    ASS_FormatLine *fmt = &track->parser_priv->event_format;
 
-    char *format = strdup(track->event_format);
-    if (!format)
-        return -1;
-    char *q = format;           // format scanning pointer
-
-    for (i = 0; i < n_ignored; ++i) {
-        NEXT(q, tname);
-    }
-
-    while (1) {
-        NEXT(q, tname);
-        if (ass_strcasecmp(tname, "Text") == 0) {
```

Comment:
Where does this check go?

Reply:
> Seems like this got dropped at some point.

Yes, it should be moved to format line parsing but vanished instead.

Reply:
Readded a warning to Event-Format parsing.